proj/hps_accel/gateware/get.py

```diff
@@ -97,6 +97,14 @@ class GetInstruction(InstructionBase):
     ]
 
     VALID_AT_RESET = {
+        Constants.REG_FILTER_0,
```

Comment:
(not blocking)

Since REGISTER_IDS and VALID_AT_RESET are now identical, should we remove VALID_AT_RESET?

Reply:
No because the next patch changes this again :-)